harvard-lts/mps-embed
# Update existing codebase dependencies for Embed

**Update existing codebase dependencies for Embed**
* * *

**JIRA Ticket**: [LTSVIEWER-216](https://jira.huit.harvard.edu/browse/LTSVIEWER-216)

# What does this Pull Request do?
Reviewed the status of dependencies of Embed and updated as necessary.

# How should this be tested?
* Spin up Embed from this branch and make sure none of the changes made to package.json or package-lock.json break it
* Updates should address the following security vulnerabilities as found by dependabot alerts (sorted by most important)
   * https://github.com/harvard-lts/mps-embed/security/dependabot/17
   * https://github.com/harvard-lts/mps-embed/security/dependabot/16
   * https://github.com/harvard-lts/mps-embed/security/dependabot/12
   * https://github.com/harvard-lts/mps-embed/security/dependabot/11
   * https://github.com/harvard-lts/mps-embed/security/dependabot/8
   * https://github.com/harvard-lts/mps-embed/security/dependabot/4
   * https://github.com/harvard-lts/mps-embed/security/dependabot/20
   * https://github.com/harvard-lts/mps-embed/security/dependabot/7
   * https://github.com/harvard-lts/mps-embed/security/dependabot/6
   * https://github.com/harvard-lts/mps-embed/security/dependabot/14
   * https://github.com/harvard-lts/mps-embed/security/dependabot/10
   * https://github.com/harvard-lts/mps-embed/security/dependabot/23
   * https://github.com/harvard-lts/mps-embed/security/dependabot/22

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz
